### PR TITLE
Re-release TF 2.7, TF 2.8 and PT 1.11

### PR DESCRIPTION
**Pull request by RadhikaB-97**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
Re-release TF2.7 inference for multi gpu support, TF 2.8 inference for multi gpu support and cve fix and PT 1.11 for PyTorch DDP distribution update.

### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release_images.yml [hunk 1]
```diff
@@ -2,182 +2,45 @@
...
-      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
+    framework: "tensorflow"
+      version: "2.7.0"
+      arch_type: "x86"
+      customer_type: "sagemaker"
+      inference:
+        device_types: ["gpu"]
```
Comment (shantanutrip) [inline release_images.yml]: Are we not releasing the cpu images for this? I read the PR description, just wanted to ensure

File: release_images.yml [hunk 2]
```diff
@@ -2,182 +2,45 @@
...
-      os_version: "ubuntu20.04"
-      cuda_version: "cu112"
-      example: False
-      disable_sm_tag: False
-      force_release: False
+      version: "2.8.0"
+      customer_type: "sagemaker"
+      arch_type: "x86"
+      inference:
+        device_types: ["cpu","gpu"]
```
Comment (shantanutrip) [inline release_images.yml]: Both cpu and gpu is being release?

File: release_images.yml [hunk 3]
```diff
@@ -2,182 +2,45 @@
...
-    framework: "autogluon"
-    version: "0.5.2"
-    arch_type: "x86"
-    training:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu112"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      cuda_version: "cu115"
```
Comment (shantanutrip) [inline release_images.yml:40]: For SM, did we release cu113 earlier or cu115?

File: release_images.yml [hunk 4]
```diff
@@ -2,182 +2,45 @@
...
-      disable_sm_tag: False
-      force_release: False
-  10:
-    framework: "pytorch"
-    version: "1.12.0"
-    arch_type: "x86"
-    customer_type: "sagemaker"
-    training:
-      device_types: [ "cpu", "gpu" ]
+      device_types: ["gpu"]
```
Comment (shantanutrip) [inline release_images.yml:37]: Just GPU?

File: release_images.yml [hunk 5]
```diff
@@ -2,182 +2,45 @@
...
-    version: "0.5.2"
-    arch_type: "x86"
-    training:
-      device_types: [ "cpu", "gpu" ]
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu112"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      cuda_version: "cu115"
+      example: True         # [Default: False] Set to True to denote that this image is an Example image
```
Comment (shantanutrip) [inline release_images.yml:41]: Why is `example` true? Are we just releasing the `example` images?

File: release_images.yml [hunk 6]
```diff
@@ -2,182 +2,45 @@
...
-      python_versions: [ "py38" ]
-      os_version: "ubuntu20.04"
-      cuda_version: "cu113"
-      example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
-      # to images being published.
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
-      # has already been published. Re-released image will have minor version incremented by 1.
+    framework: "tensorflow"
+      version: "2.7.0"
```
Comment (shantanutrip) [inline release_images.yml]: Looks like the indentation is not correct here. Can you please validate and indent the YML using https://onlineyamltools.com/validate-yaml or any other tool

File: release_images.yml (hunk 1 above)
Comment (RadhikaB-97) [reply release_images.yml]: We are releasing just TF inference SM GPU DLCs.(Ref PR - https://github.com/aws/deep-learning-containers/pull/2149 Ticket Ref- https://t.corp.amazon.com/P69594461/overview)

File: release_images.yml (hunk 2 above)
Comment (RadhikaB-97) [reply release_images.yml]: Yes, the gpu is being released for the above reason and cpu as a part of cve fix.

File: release_images.yml (hunk 3 above)
Comment (RadhikaB-97) [reply release_images.yml:40]: Changing this to cu113.

File: release_images.yml (hunk 4 above)
Comment (RadhikaB-97) [reply release_images.yml:37]: Yes, confirmed it with @viskaria.(Ticket- https://t.corp.amazon.com/P68241325/overview)

File: release_images.yml (hunk 5 above)
Comment (RadhikaB-97) [reply release_images.yml:41]: Changing that!

File: release_images.yml (hunk 6 above)
Comment (RadhikaB-97) [reply release_images.yml]: Resolved!

File: release_images.yml [hunk 7]
```diff
@@ -2,182 +2,45 @@
...
-      force_release: False   # [Default: False] Set to True to force images to be published even if the same image
+      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
       # has already been published. Re-released image will have minor version incremented by 1.
   2:
     framework: "tensorflow"
-    version: "2.9.0"
+    version: "2.8.0"
     customer_type: "sagemaker"
-    arch_type: "x86"
+    arch_type: "x86"S
```
Comment (shantanutrip) [inline release_images.yml:22]: Looks like a typo

File: release_images.yml [hunk 8]
```diff
@@ -2,182 +2,45 @@
...
+    customer_type: "sagemaker"
     inference:
-      device_types: [ "cpu", "gpu" ]
+      device_types: ["gpu"]
       python_versions: [ "py38" ]
       os_version: "ubuntu20.04"
-      cuda_version: "cu113"
       example: False        # [Default: False] Set to True to denote that this image is an Example image
-      disable_sm_tag: False # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+      disable_sm_tag: True # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
```
Comment (shantanutrip) [inline release_images.yml:14]: Why has this been set to True?

File: release_images.yml [hunk 9]
```diff
@@ -2,182 +2,45 @@
...
-    framework: "tensorflow"
-    version: "2.9.1"
-    arch_type: "x86"
-    training:
-      device_types: [ "hpu" ]
-      python_versions: [ "py38" ]
-      synapseai_version: "1.5.0"
-      os_version: "ubuntu20.04"
-      example: False
       disable_sm_tag: True  # [Default: False] This option is not used by Example images
```
Comment (shantanutrip) [inline release_images.yml:29]: Same here

File: release_images.yml [hunk 10]
```diff
@@ -2,182 +2,45 @@
 release_images:
 # Check .release_images_template.yml for reference
   1:
-    framework: "huggingface_pytorch"
-    version: "1.10.2"
+    framework: "tensorflow"
+    version: "2.7.0"
```
Comment (shantanutrip) [inline release_images.yml:6]: I think it does not have the `cuda_version` entry in it. Can you confirm if it is required or not?

**Review by shantanutrip — APPROVED** [review]

**Pull request merged**